src/Orm/Zed/Country/Persistence/SpyCountryStore.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the Spryker Commerce OS.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Orm\Zed\Country\Persistence;
+
+use Spryker\Zed\Country\Persistence\Propel\AbstractSpyCountryStore as BaseSpyCountryStore;
+
+/**
+ * Skeleton subclass for representing a row from the 'spy_country_store' table.
+ *
+ *
+ *
+ * You should add additional methods to this class to meet the
+ * application requirements. This class will only be generated as
+ * long as it does not already exist in the output directory.
+ */
+class SpyCountryStore extends BaseSpyCountryStore
```

Comment:
just a question. This looks like a part of DS. This is smth that we missed to integrate with it?